cpp/src/io/parquet/page_data.cu

```diff
@@ -763,6 +763,66 @@ inline __device__ void gpuStoreOutput(uint2 *dst, const uint8_t *src8, uint32_t
 }
 
 
+/**
+ * @brief Convert an INT96 Spark timestamp to 64-bit timestamp
+ *
+ * @param[out] dst ptr to output
+ * @param[in] src8 raw input bytes
+ * @param[in] dict_pos byte position in dictionary
+ * @param[in] dict_size size of dictionary
```

Comment:
Outdated doxygen params for the function?

Reply:
Thanks, updated.